Sources/Views/Products/Search/Scanner/ScanProductSummaryViewAdaptor.swift

```diff
@@ -125,3 +128,14 @@ private func getNovaGroup(from product: RealmOfflineProduct) -> NovaGroupView.No
     }
     return novaGroup
 }
+
+private func getEcoscoreImage(from product: RealmOfflineProduct) -> UIImage? {
+    //guard let co2Impact = product.environmentImpactLevelTags?.first else {
```

Comment:
Can this line be removed?

Reply:
Probably not, as we want to use it as fallback.